tests/installation/user_settings.pm

```diff
@@ -50,7 +50,7 @@ sub run {
             record_soft_failure('boo#1122804 - Typing issue with fullname') unless match_has_tag('inst-userinfostyped-expected-typefaces');
             $retry++;
         } while (($retry < $max_tries) && !match_has_tag('inst-userinfostyped-expected-typefaces'));
-        assert_screen('inst-userinfostyped-expected-typefaces');    # fail if mistyped
+        assert_screen('inst-userinfostyped-expected-typefaces') unless get_var('IGNORE_BSC1122804');    # fail if mistyped
```

Comment:
what if you will replace my `check_var('SLE_PRODUCT','hpc')` with this 'IGNORE_BSC1122804' ? so we will get less soft failures in case people really want to ignore this problem ?

Reply:
I am not convinced of having green when there is a mistype.

Reply:
AFAICT @schlad is more into HPC, removing it, would at least deserve pinging him... no? :)

Reply:
> AFAICT @schlad is more into HPC, removing it, would at least deserve pinging him... no? :)

ehm , I would say there is no point until I did not convince @SergioAtSUSE  :)

Reply:
> I am not convinced of having green when there is a mistype.

do you know "Test One Thing at a Time in Isolation" testing principle ? I am not saying that you "should have green" . I just saying that when you have 100 tests make all of them red or yellow because of single problem is also wrong and does not show real state of the product. there are two type of installation scenarios currently in openQA , one meant to check installation itself and they do need to become red/yellow in case any troubles but second type is for delivering images to other tests. This second type should be more error tolerant because they not meant to answer question "Does installation contain bugs?" but they answering another question "Is it possible to finish installation?" . And it is not exclusively about HPC product there are plenty of other cases when people actually get annoyed by failures like this one and even soft-failure is just a noise for them which does not change anything for state of **their** product

Reply:
I am with Anton. Installation should detect any possible issue, but other jobs that create hdd should not care about this as long as the installation performs well.

Reply:
that is the theory and then there is the practice, where people make wrappers around wrappers (grub, first_boot, reboot)

Let the test go green with a mistyped user and someone will implement another workaround in another part of the test suite to solve the same problem we are already solving in one part.

Reply:
> that is the theory and then there is the practice, where people make wrappers around wrappers (grub, first_boot, reboot, opensuse-welcome, etc)
> 
> Let the test go green with a mistyped user and someone will implement another workaround in another part of the test suite to solve the same problem we are already solving in one part.

mmm not sure that I can follow you here , my point was that we having check which not related to most of the jobs where we having it. And you mentioning something about bad coding practices which we have in this repo . How one related to another ?

Reply:
> that is the theory and then there is the practice, where people make wrappers around wrappers (grub, first_boot, reboot, opensuse-welcome, etc)
> 

I personally fully see your point!

Reply:
> "Is it possible to finish installation?" . And it is not exclusively about HPC product there are plenty of other cases when people actually get annoyed by failures like this one and even soft-failure is just a noise for them which does not change anything for state of **their** product

And they can set `IGNORE_BSC1122804=1` in the jobgroup and be happy with their lives. What we can do is that `IGNORE_BSC1122804=1` also removes the `soft-falure` and paints the test green for those who like to paint non critical errors green, be happy. 

![](https://media.giphy.com/media/y62P0aQUYLYAg/giphy.gif)